example/ts/__generated__/appQuery.graphql.ts

```diff
@@ -0,0 +1,304 @@
+/* tslint:disable */
```

Comment:
I actually want to create a TSLint rule that verifies that all properties of these fragment types are actually being used in user code to find any accidental over-fetching. This comment to disable TSLint completely is going to be interesting to deal with.

Reply:
The issue is that we can't emit TS code and be sure that we adhere to whatever tslint rules people use. It'd be great to have TSLint enabled in the emitted code, but how can we ever be sure that it is emitted in the style that the user wants?

I guess we could find all style rules and disable those?

Reply:
I don’t think we need to attempt that, disabling the linter wholesale is imo the right decision (which is what relay-compiler does as well).

I was just kinda thinking out loud about how to deal with that, I’ll figure something out 👍